hikari/applications.py

```diff
@@ -613,6 +613,88 @@ def make_cover_image_url(self, *, ext: str = "png", size: int = 4096) -> typing.
             file_format=ext,
         )
 
+    async def fetch_guild(self) -> typing.Optional[guilds.RESTGuild]:
+        """Fetch the guild this application is linked to if sold on Discord.
+
+        Parameters
+        ----------
+        guild : hikari.snowflakes.SnowflakeishOr[hikari.guilds.PartialGuild]
+            The guild to fetch. This can be the object
+            or the ID of an existing guild.
```

Comment:
This is not needed.

Reply:
Oops yeah my bad. Basically copied the docstring from rest.fetch_guild